gef-ui/server/constants/index.js

```diff
@@ -74,6 +74,13 @@ const DATE_FORMAT = {
   COVER: 'MMMM d, yyyy',
 };
 
+const USER_ROLES = {
```

Comment:
Nice change

Reply:
I had a quick look at there are a loootttt of use cases of this in different route files and a couple of other places. Might be worth just adding a TODO and creating a jira tech debt ticket to move to using these constants. Don't want to hold up this ticket for something like that :)